docs/rst/notes/forthcoming_version.rst

```diff
@@ -5,3 +5,11 @@
 ###################
 Forthcoming Version
 ###################
+
+This release will include the following **Recording features**:
+
+* Support configuring logging messages with a YAML :ref:`configuration file <recorder_specs_logging>`.
```

Comment:
New configuration option (``logging``) to configure the :ref:`Logs <recorder_specs_logging>`.